cli/providers/aws/translate.go

```diff
@@ -64,7 +64,7 @@ func Translate(s *service.Service, e *environment.Environment, config *aws.Confi
 	loadIAM(t, s, e)
 	loadLogGroup(t, s, e)
 	ports := s.GetPorts()
-	if len(ports) > 0 {
+	if len(ports) > 0 && e.Provider.HostedZone != "" {
```

Comment:
should this be e.Root.HostedZone?

Reply:
No. There are two use cases, user provided DNS, and our default DNS